### DISCUSSION TITLE
docker最新版无法挂载各类盘，阿里、天翼、webdav

### DISCUSSION THREAD

**Discussion by sontjer** [AlistGo/alist#1623] [Q&A]
docker最新版无法挂载各类盘，阿里、天翼、webdav
提示如下：Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

**Comment by xhofe** [comment]
unique mount path 很难理解吗

**Reply by sontjer** [reply]
大佬，谢谢回复！设置并没有变，之前的版本没有要另外设置什么mount path.

**Reply by sontjer** [reply]
> 我试了一下数据库结构都变了，需要重新设置

请问你重新设置可以正常运行吗？

**Reply by lollyhero** [reply]
重新设置就可以了 我试过了

**Reply by sontjer** [reply]
> 重新设置就可以了 我试过了

All done after restarting container, thanks bro.

**Reply by Jim-CN** [reply]
我的alist docker从2.6.3自动更新latest就无法运行了，web 页面报错：failed get storage: can't find storage with rawPath: /

**Reply by AlleNPooh** [reply]
有人知道怎么解决嘛？而且我的密码忘记了，重置密码没法用

**Reply by chen8945** [reply]
> 有人知道怎么解决嘛？而且我的密码忘记了，重置密码没法用

**docker exec -it alist ./alist admin**
可以查看密码，如果你的容器名字不是 alist，请替换命令中相应内容

**Comment by sontjer** [comment]
docker logs alist

在 2022年9月11日星期日，AllenPooh ***@***.***> 写道：

> 有人知道怎么解决嘛？而且我的密码忘记了，重置密码没法用
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/alist-org/alist/discussions/1623#discussioncomment-3615844>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKIJKKHROH2SWB3K4RLVKS3V5TNVTANCNFSM6AAAAAAQIRAZVA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Sincere regards,

**Reply by AlleNPooh** [reply]
root@UNRAID-5070:~# docker logs AList
INFO[2022-09-10 18:53:02] reading config file: data/config.json        
INFO[2022-09-10 18:53:02] load config from env with prefix:            
INFO[2022-09-10 18:53:02] init logrus...                               
INFO[2022-09-10 18:53:03] start server @ 0.0.0.0:5244                  
ERRO[2022-09-10 18:53:03] failed to init aria2 client: Post "http://localhost:6800/jsonrpc": dial tcp 127.0.0.1:6800: connect: connection refused
failed get aria2 version
github.com/alist-org/alist/v3/internal/aria2.InitAria2Client
        /app/internal/aria2/aria2.go:33
github.com/alist-org/alist/v3/internal/aria2.InitClient
        /app/internal/aria2/aria2.go:23
github.com/alist-org/alist/v3/internal/bootstrap.InitAria2.func1
        /app/internal/bootstrap/aria2.go:10
runtime.goexit
        /usr/lib/go/src/runtime/asm_amd64.s:1594

**Reply by AlleNPooh** [reply]
这是日志里的所有东西

**Reply by jile1997** [reply]
我也遇到了这个问题 你解决了吗

**Reply by jile1997** [reply]
找到说明了 可以看下这个链接 https://alist.nn.ci/zh/faq/howto.html

**Reply by HarryChong** [reply]
能解释一下这个链接怎么解决么，我也遇到同样的问题，是不是配置文件的问题？

**Comment by magiccode1412** [comment]
是不是挂载路径都填了一样的，我之前没改，都填了“/”，后来才发现忘记改了，挂载路径得不一样
